docs/extend/extend-cli/cli-extending-a-plugin.md

```diff
@@ -10,77 +11,85 @@ This tutorial demonstrates how to extend the plug-in that is bundled with this s
 
-```typescript
-import { ITodo } from "./doc/ITodo";
-import { RestClient, AbstractSession, ImperativeExpect, Logger } from "@zowe/imperative";
+2. Edit the `Typicode.ts` file to contain the following code: **[correct?]**
 
-export class Typicode {
+    ```typescript
+    import { ITodo } from "./doc/ITodo";
+    import { RestClient, AbstractSession, ImperativeExpect, Logger } from "@zowe/imperative";
 
-    public static readonly TODO_URI = "/todos";
+    export class Typicode {
 
-    public static getTodos(session: AbstractSession): Promise<ITodo[]> {
-        Logger.getAppLogger().trace("Typicode.getTodos() called");
-        return RestClient.getExpectJSON<ITodo[]>(session, Typicode.TODO_URI);
-    }
+        public static readonly TODO_URI = "/todos";
+
+        public static getTodos(session: AbstractSession): Promise<ITodo[]> {
+            Logger.getAppLogger().trace("Typicode.getTodos() called");
+            return RestClient.getExpectJSON<ITodo[]>(session, Typicode.TODO_URI);
+        }
 
-    public static getTodo(session: AbstractSession, id: number): Promise<ITodo> {
-        Logger.getAppLogger().trace("Typicode.getTodos() called with id " + id);
-        ImperativeExpect.toNotBeNullOrUndefined(id, "id must be provided");
-        const resource = Typicode.TODO_URI + "/" + id;
-        return RestClient.getExpectJSON<ITodo>(session, resource);
+        public static getTodo(session: AbstractSession, id: number): Promise<ITodo> {
+            Logger.getAppLogger().trace("Typicode.getTodos() called with id " + id);
+            ImperativeExpect.toNotBeNullOrUndefined(id, "id must be provided");
+            const resource = Typicode.TODO_URI + "/" + id;
+            return RestClient.getExpectJSON<ITodo>(session, resource);
+        }
     }
-}
 
-```
+    ```
 
-The `Typicode` class provides two programmatic APIs, `getTodos` and `getTodo`, to get an array of `ITodo` objects or a specific
-`ITodo` respectively. The Node.js APIs use `@zowe/imperative` infrastructure to provide logging, parameter validation,
-and to call a REST API. See the [Imperative CLI Framework documentation](https://github.com/zowe/imperative/wiki) for more information.
+    The `Typicode` class provides two programmatic APIs, `getTodos` and `getTodo`, to get an array of `ITodo` objects or a specific `ITodo`, respectively.
+
+    The Node.js APIs use `@zowe/imperative` infrastructure to provide logging, parameter validation, and to call a REST API. See the [Imperative CLI Framework documentation](https://github.com/zowe/imperative/wiki) for more information. **[this repo is archived. what else can we link to here?]**
 
 #### Exporting interface and programmatic API for other Node.js applications
+**[Feels like there's a missing word in this ^^ header. Can we say "the" interface and programmatic ...?]**
```

Comment:
@adam-wolfe please respond

Reply:
Yes, I would change it to "Exporting _the_ interface and programmatic API for other ..."